backup/drive.go

```diff
@@ -359,6 +364,15 @@ func (p *GoogleDriveProvider) DownloadBackupFiles(nodeID, backupID string) ([]st
 	return downloaded, nil
 }
 
+// Parse is responsible for parsing a string to a time.Time.
+func (p *GoogleDriveProvider) ParseTime(s string) (time.Time, error) {
```

Comment:
I have noticed this method is not in use. If so let's remove it.

Reply:
Removed in latest commit